camkes/templates/component.common.c

```diff
@@ -498,7 +498,12 @@ static void CONSTRUCTOR(CAMKES_SYSCALL_CONSTRUCTOR_PRIORITY+1) init(void) {
     /*- elif options.debug_fault_handlers and loop.last -*/
         /*- do thread_names.__setitem__(tcb, "fault_handler") -*/
         /*- do _tcb.__setattr__('prio', 255) -*/
-        /*- do _tcb.__setattr__('affinity', options.default_affinity) -*/
+        /*- set thread_affinity = configuration[me.name].get("affinity") -*/
```

Comment:
`.get()` takes an optional second argument that can be used to provide a default if the lookup fails which looks like it could make some of these macro option passing sequences shorter to write.

Reply:
Done